src/schemas/runtimestrain.schema.json

```diff
@@ -63,8 +63,33 @@
       "type": "boolean"
     },
     "package": {
-      "description": "Name of the action package that renders this strain.",
-      "type": "string"
+      "oneOf": [
+        {
+          "description": "Name of the action package that renders this strain.",
+          "type": "string",
+          "pattern": "^[^/]*$",
+          "examples": [
+            "10e05c072b108ac9533c6206c6eaa651901ca29b"
```

Comment:
I think we don't use this format anymore. it was always `namespace/package`

Reply:
True, but the tests still have it and I did not want to break compatibility.